mailer.js

```diff
@@ -96,6 +122,14 @@ module.exports = function (log) {
     return parts.join(', ')
   }
 
+  Mailer.prototype._constructLocationString = function (location) {
+    return constructLocationString(location)
+  }
+
+  Mailer.prototype._constructLocalTimeString = function (timestamp, locale) {
+    return constructLocalTimeString(timestamp, locale)
```

Comment:
The function signature is `function constructLocalTimeString (timeZone, locale)` but here we're passing a first argument called `timestamp` rather than `timeZone`...is one of these mis-named?